packages/odata-common/src/filter/unary-filter.ts

```diff
@@ -5,7 +5,7 @@ import type { Filterable } from './filterable';
 type UnaryFilterOperator = 'not';
 
 /**
- * @internal
+ * PLEASE REVIEW.
```

Comment:
Class representing a unary filter which is a filter containing just a single argument e.g. `not`